src/main/java/org/sonar/plugins/resharper/ReSharperPlugin.java

```diff
@@ -29,6 +29,7 @@
 
 public class ReSharperPlugin extends SonarPlugin {
 
+  public static final String PROJECT_NAME_SONAR_PROPERTY_KEY = "sonar.projectName";
```

Comment:
it should be `sonar.resharper.projectName` to make it clear that it is a property for the SonarQube ReSharper plugin